packages/design-system/src/Header/Header.tsx

```diff
@@ -0,0 +1,42 @@
+import { HTMLAttributes } from 'react';
+import { cn } from '@soup/utils';
+
+export interface HeaderProps extends HTMLAttributes<HTMLElement> {}
```

Comment:
이 부분은 따로 추가하는 속성이 없다면 `type`으로 사용해도 좋을 것 같아요!
```suggestion
import { HTMLAttributes } from 'react';
import { cn } from '@soup/utils';

export type HeaderProps = HTMLAttributes<HTMLElement>;
```

Reply:
`HeaderProps`가 객체 타입을 정의하고, 추후 확장가능성을 생각해서 직관적인 `interface`를 사용하였어요. 전체적인 코드 스타일과도 일관성을 유지하기 위해 `interface`를 사용했는데 혹시 어떻게 생각하실까요?

Reply:
빈 집합의 `interface`를 보고 제안드렸는데, 추후 확장 가능성이 있다면 저렇게 유지해도 괜찮을 것 같아요! 확장성 관련해서는 둘 다 타입으로 만드는 방법도 있긴 하지만, `interface`를 선호하신다면 지금 그대로도 괜찮을 것 같습니다 👍